### PR TITLE
Update frontend config to match b20437b3

### PR DESCRIPTION
**Pull request by erenon**

### CODE REVIEW

File: bare/config/frontend.jsonnet [hunk 1]
```diff
@@ -8,7 +8,7 @@ local common = import 'common.libsonnet';
     authenticationPolicy: { allow: {} },
   }],
   schedulers: {
-    'local': { address: 'localhost:8982' },
+    'local': { endpoint: { address: 'localhost:8982' } },
```
Comment (EdSchouten) [inline bare/config/frontend.jsonnet:11]: I think you will need to change the key of this entry to be `''`. The default behavior is to now strip the part of the instance name that is provided here.

Comment (erenon) [reply bare/config/frontend.jsonnet]: fixed.

File: kubernetes/config/storage.yaml [hunk 2]
```diff
@@ -30,7 +30,7 @@ data:
         listenAddresses: [':8981'],
         authenticationPolicy: { allow: {} },
       }],
-      allowAcUpdatesForInstances: ['bb-event-service', 'remote-execution'],
+      allowAcUpdatesForInstanceNamePrefixes: ['bb-event-service', 'remote-execution'],
```
Comment (EdSchouten) [inline kubernetes/config/storage.yaml:33]: If you want, you can change this to use the empty string as well:
```
allowAcUpdatesForInstanceNamePrefixes: [''],
```

**Review by EdSchouten — COMMENTED** [review]
Shouldn't we also bump image names/tags somewhere?

File: kubernetes/config/storage.yaml (hunk 2 above)
Comment (erenon) [reply kubernetes/config/storage.yaml]: done

**Comment by erenon** [comment]
> Shouldn't we also bump image names/tags somewhere?

And the version perhaps. Is there a tool to do that?

**Comment by EdSchouten** [comment]
There is no automation for that (yet). Be sure to just copy-paste the latest tags from Docker Hub.

**Comment by erenon** [comment]
Image versions bumped.

**Comment by olib963** [comment]
I believe you missed the image bump for [kubernetes](https://github.com/buildbarn/bb-deployments/blob/fec381e782696b64b70ece70eb7a1b727e52c437/kubernetes/storage.yaml#L20). I had to faff around for a while to figure out why it was failing to parse the config. The current setup means that the action cache rejects everything.

Worth investing in some automated testing for this repo? Or perhaps more trouble than it is worth?

**Pull request merged**